src/integration-tests/src/showcase.rs

```diff
@@ -0,0 +1,168 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::Error;
+use crate::Result;
+use gax::options::RequestOptionsBuilder;
+use showcase::model::compliance_data::LifeKingdom;
+use std::process::Stdio;
+use std::time::Duration;
+use tokio::process::Command;
+
+const SHOWCASE_NAME: &str = "github.com/googleapis/gapic-showcase/cmd/gapic-showcase@latest";
```

Comment:
nit observation: using `latest` in the CI.

No changes required, just point it out.

Reply:
Fixed.

Reply:
Good call btw.